tools/compiletest/src/main.rs

```diff
@@ -205,6 +226,15 @@ pub fn run_tests(config: Config) {
         make_tests(c, &mut tests);
     }
 
+    if config.dry_run {
+        println!("Number of Tests: {}", tests.len());
+        for test in tests {
+            let ignore = if test.desc.ignore ^ config.run_ignored { "IGNORE" } else { "" };
+            println!(" - {} {}", test.desc.name.as_slice(), ignore);
```

Comment:
Most of the output coming from compiletest doesn't print capital letters. Does this look similar to running without `--quiet`? Also, shouldn't `--quiet` or `--verbose` conflict with this flag?

Reply:
I should probably just use `logv` here to ensure `verbose` and `quiet` are being respected. I'll change that.

I don't think there should be any conflict though.